_posts/2016-09-16-puppet-corosync-reaches-5.0.0.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: Puppet-Corosync reaches 5.0.0
+date: 2016-09-16
+github_username: roidelapluie
+twitter_username: roidelapluie
+---
+
+This is a great moment for all the [Vox Pupuli's puppet-corosync][a] puppet module.
+The [5.0.0 major release][b] is there, with lots of new features and improvements.
+
+The puppet-corosync module was written as a Puppetlabs module until 2014, until
+its move to the Puppet Community (former name of Vox Pupuli). That move was a
```

Comment:
should we rearrange this?
`move to Vox Pupuli (former name Puppet Community).`
I am not sure about the correct sentence construction in English, in German you start with the current thing and name the older one at the end.